api/control/v1alpha1/types.go

```diff
@@ -349,6 +349,10 @@ type Destination struct {
 }
 
 type BindingStatus struct {
+	// Currently Conditions is by design to be copied to BindingPolicy's Conditions.
+	// +optional
+	Conditions []BindingPolicyCondition `json:"conditions"`
```

Comment:
It is Kubernetes design style to put `,omitempty` in the json tag.

Reply:
Done. Also fixed same thing in `type BindingPolicyStatus`.